src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -506,7 +555,7 @@ private IVariableInformation[] ExpandVisualized(IVariableInformation variable)
             }
             foreach (var i in expandType.Items)
             {
-                if (i is ItemType)
+                if (i is ItemType && variable.Name != ResourceStrings.MoreView) // we do not want to display Size/Count when expanding the "[More...]" node
```

Comment:
Someone could named their visualized variable to be `[More...]`.

It would be better to check to see if it is a `PaginatedVisualizerWrapper` type.